Misc/NEWS.d/next/Library/2021-04-22-04-12-13.bpo-43817.FQ-XlH.rst

```diff
@@ -0,0 +1,6 @@
+Add inspect.get_annotations(o), which safely retrieves and returns the
+annotations for o.  Also un-stringizes the annotations if they are
+stringized.
+
+Modified inspect.signature() to use get_annotations() to retrieve
+annotations.
```

Comment:
```suggestion
Add ``inspect.get_annotations(o)``, which safely retrieves and returns the
annotations for ``o``.  Also un-stringizes the annotations if they are
stringized.

Modified ``inspect.signature()`` to use ``get_annotations()`` to retrieve
annotations.
```

Reply:
Thanks!  I'm not sure I'm sophisticated enough at Github to merge your suggestion using Github itself, but I'll make that change when I update the PR.